### PR TITLE
[Feature:Submission] Instructor UI Poll Edit and Delete Buttons to Icons

### PR DESCRIPTION
**Pull request by listeph**
### Please check if the PR fulfills these requirements:

* [ ] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Fixes most recent comment on #6036 (doesn't close the whole issue!)

> The "Delete" button the instructor UI is rarely used, and can take up less room... replace it with a trash can. The "Edit poll" button the instructor UI could be replaced with the pencil icon. Maybe put this to the left of the poll title?
Let's standardize the styling of this page to make it more similar to the all gradeables page instructor view.

old

### What is the new behavior?

So changes are that
- the delete and edit buttons are replaced by icons to the left of name
- deleting now must be confirmed in a dialog box
- unnecessary header titles have been removed
- name is now left justified in its column, to match gradeables navigation page

Screen Shot 2020-12-09 at 1 33 22 PM

### Other Information

I don't know what to return from deletePoll() now, since I no longer use the redirect response; I just reload using the javascript AJAX function. I was thinking of just leaving it the way it is? (The function isn't used anywhere else other than this twig file)

### CODE REVIEW

**Review by shailpatels — APPROVED** [review]
Changes look good and works as described

**Comment by bmcutler** [comment]
Please fix merge conflicts.
And also we'll wait to merge this until the delete button gets an "are you sure" popup.

**Review by vikrsri — APPROVED** [review]
Changes seem to work as proposed, delete button popup message looks good.

**Comment by bmcutler** [comment]
@listeph @vikrsri 
Uh, oh, merge conflicts again.  Can one of you please resolve?

**Comment by vikrsri** [comment]
Merge conflicts have been resolved

File: site/app/templates/polls/AllPollsPageInstructor.twig [hunk 1]
```diff
@@ -12,28 +12,33 @@
...
-            <col style="width: 26%">
+            <col style="width: 30%">
+            <col style="width: 14%">
+            <col style="width: 14%">
+            <col style="width: 14%">
+            <col style="width: 14%">
+            <col style="width: 14%">
             <thead>
                 <tr>
+                    <th scope="col"></th>
```
Comment (listeph) [inline site/app/templates/polls/AllPollsPageInstructor.twig:23]: I'm confused about this extra column

File: site/app/templates/polls/AllPollsPageInstructor.twig [hunk 2]
```diff
@@ -12,28 +12,33 @@
...
                     <tr>
-                        <td align="left"> 
+                        <td align="left">
                             <a class="fas fa-pencil-alt black-btn" tabindex="0" href="{{base_url}}/editPoll/{{ poll.getID() }}" title="edit poll {{ poll.getName() }}" aria-label="edit poll {{ poll.getID() }}">
                                 <input type="hidden" name="poll_id" value="{{ poll.getID() }}"/>
                             </a>
                             <a class="fas fa-trash black-btn" tabindex="0" href='javascript:newDeletePollForm("{{ poll.getID() }}", "{{ poll.getName() }}","{{ base_url }}");' title="delete poll {{ poll.getName() }}" aria-label="delete poll {{ poll.getID() }}"></a>
                             {{ poll.getName() }}
                         </td>
+                        <td></td>
```
Comment (listeph) [inline site/app/templates/polls/AllPollsPageInstructor.twig:41]: and this

File: site/app/templates/polls/AllPollsPageInstructor.twig [hunk 3]
```diff
@@ -120,27 +127,36 @@
...
-                        <td align="left"> 
+                        <td align="left">
                             <a class="fas fa-pencil-alt black-btn" tabindex="0" href="{{base_url}}/editPoll/{{ poll.getID() }}" title="edit poll {{ poll.getName() }}" aria-label="edit poll {{ poll.getID() }}">
                                 <input type="hidden" name="poll_id" value="{{ poll.getID() }}"/>
                             </a>
                             <a class="fas fa-trash black-btn" tabindex="0" href='javascript:newDeletePollForm("{{ poll.getID() }}", "{{ poll.getName() }}", "{{ base_url }}");' title="delete poll {{ poll.getName() }}" aria-label="delete poll {{ poll.getID() }}"></a>
                             {{ poll.getName() }}
                         </td>
                         <td> {{ poll.getReleaseDate() }} </td>
+                        <td></td>
```
Comment (listeph) [inline site/app/templates/polls/AllPollsPageInstructor.twig:157]: and this

**Review by vikrsri — APPROVED** [review]
Changes look good and ready to merge.

**Pull request merged**